src/components/Navbar.tsx

```diff
@@ -30,8 +29,8 @@ const Navbar = () => {
         { label: (<Link  to="/peers">Peers</Link>), key: '/peers' },
         { label: (<Link  to="/add-peer">Add Peer</Link>), key: '/add-peer' },
         { label: (<Link  to="/setup-keys">Setup Keys</Link>), key: '/setup-keys' },
-        /*{ label: (<Link  to="/acls">Access Control</Link>), key: '/acls' },
-        { label: (<Link  to="/activity">Activity</Link>), key: '/activity' },*/
+        { label: (<Link  to="/acls">Access Control</Link>), key: '/acls' },
```

Comment:
Codacy has a fix for the issue: [Replace `········{·label:·(<Link··to="/acls">Access·Control</Link>),·key:·'/acls'` with `····{·label:·<Link·to="/acls">Access·Control</Link>,·key:·"/acls"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
    { label: <Link to="/acls">Access Control</Link>, key: "/acls" },
```